SegmentEditorDrawTube/SegmentEditorDrawTubeLib/SegmentEditorEffect.py

```diff
@@ -168,7 +168,7 @@ def setMRMLDefaults(self):
   def updateGUIFromMRML(self):
     if self.segmentMarkupNode:
       self.cancelButton.setEnabled(self.segmentMarkupNode.GetNumberOfFiducials() is not 0)
-      self.applyButton.setEnabled(self.segmentMarkupNode.GetNumberOfFiducials() >= 3)
+      self.applyButton.setEnabled(self.segmentMarkupNode.GetNumberOfDefinedControlPoints() >= 3)

```

Comment:
This is not Slicer 4.10 compatible as this method wasn't available in Slicer until https://github.com/Slicer/Slicer/commit/4b6c86e92254a6881aeac5adaa322525f05f6feb.  Same goes for other uses of `GetNumberOfDefinedControlPoints()` in this PR.
```python
      if slicer.app.majorVersion >= 5 or (slicer.app.majorVersion == 4 and slicer.app.minorVersion >= 11):
        self.cancelButton.setEnabled(self.segmentMarkupNode.GetNumberOfDefinedControlPoints() is not 0)
        self.applyButton.setEnabled(self.segmentMarkupNode.GetNumberOfDefinedControlPoints() >= 3)
      else:
        self.cancelButton.setEnabled(self.segmentMarkupNode.GetNumberOfFiducials() is not 0)
        self.applyButton.setEnabled(self.segmentMarkupNode.GetNumberOfFiducials() >= 3)
```